### PR TITLE
update dependencies

### PR DESCRIPTION
**Pull request by ramitos**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ const {
 } = process.env;
 
 const server = Hapi.server({
+  compression: {
+    minBytes: 1
```
Comment (cjihrig) [inline index.js]: I think some performance analysis tools prefer for **everything** to be compressed, but the hapi default (1024) should be fine here unless we have benchmark numbers. For small payloads I think the extra CPU hit is not worth it, and the uncompressed message can be sent in the same number of packets.

@geek what do you think here?

Comment (ramitos) [reply index.js]: @cjihrig I agree 😉

**Comment by ramitos** [comment]
@cjihrig @geek please don't merge yet. I want to bundle the Images console (temp) here too.

**Comment by ramitos** [comment]
@cjihrig updated

**Pull request merged**